apps/api/src/.env.test

```diff
@@ -110,3 +110,5 @@ IS_USE_MERGED_DIGEST_ID_ENABLED=true
 
 HUBSPOT_INVITE_NUDGE_EMAIL_USER_LIST_ID=
 HUBSPOT_PRIVATE_APP_ACCESS_TOKEN=
+
+NOVU_ENTERPRISE=true
```

Comment:
Would it make sense to include this in the repository? The tests expect this flag if I'm not mistaking, so it makes sense to me to keep this in